blog/temporal-introduction.md

```diff
@@ -0,0 +1,420 @@
+assume most people who have used Rust would be familiar with `from`. But
+this isn't Rust's friendly `From` trait. No, this `from` is a behemoth
+method that takes a `JsValue` and automagically gives you back the
+built-in that you'd like or throws. That's right! Give it a string, give
+it a property bag, give it another built-in; `from` will figure it out
+for you!
+
+Simple, right?
+
+Well we're pleased to announce that `temporal_rs` won't be supporting
+that! ... At least in one method.
+
+Again, the goal of `temporal_rs` is to implement the specification to
+the highest degree possible and where it can't implement the
+specification like in the case of a `from<JsValue>`, it provides methods
+to support the implementation of Temporal in engines.
+
+Let's take a look at some valid uses of `from` in JavaScript to
+construct a `PlainDate`.
+
+```js
+// Create a `PlainDateTime`
+const pdt = new Temporal.PlainDateTime(2025, 1, 1);
+// We can use the `PlainDateTime` (`ZonedDateTime` / `PlainDate` are also options).
+const pd_from_pdt = Temporal.PlainDate.from(pdt);
+// We can use a string.
+const pd_from_string = Temporal.PlainDate.from("2025-01-01");
+// We can use a property bag.
+const pd_from_property_bag = Temporal.PlainDate.from({
+  year: 2025,
+  month: 1,
+  day: 1,
+});
+```
+
+It turns out that the above operations is all that needs to be
+implemented by `temporal_rs` in order to support a Temporal
+implementation, we just need to implement `From<PlainDateTime>`,
+`From<ZonedDateTime>`, `FromStr`, and `From<JsObject>` ... oh. Did I
+mention `JsObject`, like `JsValue`, is implementation defined as well?
+
+This is where `temporal_rs`'s Partial API comes in.
+
+It turns out that while property bags in JavaScript can have various
+fields set, there is still a general shape for the fields that can be
+provided and validated in Temporal.
+
+So in order to support this in `temporal_rs`, a "partial" component
+exists for each of the components that can then be provided to that
+components `from_partial` method.
+
+With this, we have fully implemented support for the `from` method in
+`temporal_rs`:
+
+```rust
+use core::str::FromStr;
+use temporal_rs::{PlainDate, PlainDateTime, partial::PartialDate};
+let pdt = PlainDateTime::try_new_iso(2025, 1, 1)?;
+// We can use the `PlainDateTime` (`ZonedDateTime` / `PlainDate` are also options).
+let pd_from_pdt = PlainDate::from(pdt);
+// We can use a `str`.
+let pd_from_string = PlainDate::from_str("2025-01-01")?;
+// We can use a `PartialDate`.
+let pd_from_partial = PlainDate::from_partial(PartialDate {
+    year: Some(2025),
+    month: Some(1),
+    day: Some(1),
+    ..Default::default()
+});
+```
+
+**NOTE:** there may be updates to `PartialDate` in the future (see
+[boa-dev/temporal #349](https://github.com/boa-dev/temporal/issues/349)
+for more information).
+
+## Elephant in the room: time zones
+
+So far we have not dicussed time zones, and -- suprise! -- we aren't
```

Comment:
```suggestion
So far we have not discussed time zones, and -- surprise! -- we aren't
```

Reply:
Oops lol I've looked at this line so many times too.